app/models/achievement.rb

```diff
@@ -0,0 +1,9 @@
+class Achievement < ApplicationRecord
+  has_many :achievement_users
+  has_many :users, through: :achievement_users
+
+  validates :name, :points, presence: true
+  def to_s
+  	name
+  end
+end
```

Comment:
Oh, and you should set up your editor to add a newline at the end of file when you save.

Reply:
Everything should be fixed now.